src/provider/model/AppDirectory.ts

```diff
@@ -33,32 +26,16 @@ export class AppDirectory extends AsyncInit {
 
     protected async init(): Promise<void> {
         await this._configStore.initialized;
-
-        if (process.env.NODE_ENV === 'development') {
-            // Set the application directory to our local copy during development.  In production it will be an empty string.
-            this._configStore.config.add({level: 'desktop'}, {applicationDirectory: DEV_APP_DIRECTORY_URL});
-        }
-
-        this.updateUrl(this._configStore.config.query({level: 'desktop'}).applicationDirectory);
-
-        const watch = new MaskWatch(this._configStore.config, {applicationDirectory: true});
-
-        watch.onAdd.add((rule: ScopedConfig<ConfigurationObject>, source: Scope) => {
-            this.updateUrl(this._configStore.config.query({level: 'desktop'}).applicationDirectory);
```

Comment:
This was a sort of workaround if you remember before.  We no longer need it because we are setting the application directory elsewhere (no longer checking if we are in dev). Also the application directory location cannot change once the service has started so we don't need to monitor it.